src/components/elements/market-tab/index.tsx

```diff
@@ -27,24 +23,29 @@ const MarketTab = () => {
         <>
             <MarketsContainer>
                 {market_buttons.map((marketItem) => (
-                    <MarketButton
-                        selected={marketItem.market_name === selected_market}
-                        key={marketItem.id}
-                        onClick={() => {
-                            onMarketButtonClick(marketItem.market_name)
-                        }}
-                    >
-                        <MarketIcon
-                            src={
-                                marketItem.market_name === selected_market
-                                    ? marketItem.selected_src
-                                    : marketItem.src
-                            }
-                        />
-                        <TabStyledHeader type="paragraph-2" as="p">
-                            <Localize translate_text={marketItem.button_text} />
-                        </TabStyledHeader>
-                    </MarketButton>
+                    <>
+                        <MarketButton
+                            selected={marketItem.market_name === selected_market}
+                            key={marketItem.id}
```

Comment:
While we have exclusive `market_name` we can remove `id` from data at all
```suggestion
                            key={marketItem.market_name}
```